unattended_installer/cert_tool/certFunctions.sh

```diff
@@ -91,23 +91,40 @@ function cert_generateCertificateconfiguration() {
         IP.1 = cip
 	EOF
 
-    conf="$(awk '{sub("CN = cname", "CN = '"${1}"'")}1' "${cert_tmp_path}/${1}.conf")"
-    echo "${conf}" > "${cert_tmp_path}/${1}.conf"
 
     isIP=$(echo "${2}" | grep -P "^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$")
     isDNS=$(echo "${2}" | grep -P "^[a-zA-Z0-9][a-zA-Z0-9-]{1,61}[a-zA-Z0-9](?:\.[a-zA-Z-]{2,})+$" )
 
-    if [[ -n "${isIP}" ]]; then
-        conf="$(awk '{sub("IP.1 = cip", "IP.1 = '"${2}"'")}1' "${cert_tmp_path}/${1}.conf")"
-        echo "${conf}" > "${cert_tmp_path}/${1}.conf"
-    elif [[ -n "${isDNS}" ]]; then
-        conf="$(awk '{sub("CN = cname", "CN =  '"${2}"'")}1' "${cert_tmp_path}/${1}.conf")"
-        echo "${conf}" > "${cert_tmp_path}/${1}.conf"
-        conf="$(awk '{sub("IP.1 = cip", "DNS.1 = '"${2}"'")}1' "${cert_tmp_path}/${1}.conf")"
-        echo "${conf}" > "${cert_tmp_path}/${1}.conf"
+    conf="$(awk '{sub("CN = cname", "CN = '"${1}"'")}1' "${cert_tmp_path}/${1}.conf")"
+    echo "${conf}" > "${cert_tmp_path}/${1}.conf"
+
+    if [ "${#@}" -gt 2 ]; then
+        sed -i '/IP.1/d' "${cert_tmp_path}/${1}.conf"
+        for (( i=2; i<=${#@}; i++ )); do
+            isIP=$(echo "${!i}" | grep -P "^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$")
+            isDNS=$(echo "${!i}" | grep -P "^[a-zA-Z0-9][a-zA-Z0-9-]{1,61}[a-zA-Z0-9](?:\.[a-zA-Z-]{2,})+$" )
+            j=$((i-1))
+            if [ "${isIP}" ]; then
+                printf '%s\n' "        IP.${j} = ${!i}" >> "${cert_tmp_path}/${1}.conf"
+            elif [ "${isDNS}" ]; then
+                printf '%s\n' "        DNS.${j} = ${!i}" >> "${cert_tmp_path}/${1}.conf"
+            else
+                common_logger -e "Invalid IP or DNS ${!i}"
```

Comment:
Why use 2 different messages for the same error?

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/1770/commits/4ea226db5e29ede078f86b3b799eb04b1a5bac84